src/test/java/org/eclipse/basyx/testsuite/regression/submodel/metamodel/map/TestSubmodel.java

```diff
@@ -85,4 +87,27 @@ public void testCreateAsFacadePropertyList() {
 	protected ISubmodel getSubmodel() {
 		return submodel;
 	}
+	
+	@Test(expected = IdShortDuplicationException.class)
+	public void checkForExceptionWithDuplicateIdShortInSubmodel() {
+		Map<String, Object> faultySubmodel = createSubmodelWithDuplicateIdShortProperties();
+
+		Submodel.createAsFacade(faultySubmodel);
+	}
+
+	private Map<String, Object> createSubmodelWithDuplicateIdShortProperties() {
+		Property property1 = new Property("testProp", 5);
```

Comment:
Please extract the common IdShort in a String variable called, e.g., duplicateIdShort